doc/guides/porting/shields.rst

```diff
@@ -71,16 +71,19 @@ introduced overriding node element:
 Board specific shield configuration
 -----------------------------------
 
-If modifications are needed to fit a shield to a particular board, you can
-override a shield description for a specific board by adding board-overriding
-files to a shield, as follows:
+If modifications are needed to fit a shield to a particular board or board
+revision, you can override a shield description for a specific board by adding
+board or board revision overriding files to a shield, as follows:
```

Comment:
```suggestion
board or board revision overlay files to a shield, as follows:
```

Reply:
The original description already used `override`, although that doesn't mean that term is correct.

For DTS, the overlay term is correct, but for Kconfig, then the correct term is actually a fragment with the settings you want to override.
https://docs.zephyrproject.org/latest/application/index.html?highlight=fragment#important-build-system-variables

I guess this is one of the places where it's hard to find a common name that is precise for both cases.

Reply:
On my side, I'm fine with current proposal.

Reply:
Fair enough, I didn't pay close attention and missed that there is a `<board>_<revision>.conf` file also.